Modules/_blake2/blake2s_impl.c

```diff
@@ -18,7 +18,7 @@
 #endif
 
 #include "Python.h"
```

Comment:
Are the changes in this file accidental?

Reply:
This is a generated file. After regenerating all I got this.